src/components/BankTransactionRow/bankTransactionRow.scss

```diff
@@ -10,3 +10,13 @@
   overflow: hidden;
   width: 100%;
 }
+
+.Layer__BankTransactionRow__Description {
+  width: 40%;
+}
+
+@container (max-width: 760px) {
+  .Layer__BankTransactionRow__Description {
+    width: 100%;
+  }
```

Comment:
Nit, PR review experience: When I use magic numbers like `40%` and especially when I am doing FE work, I usually like to include a screen recording of how this media query handles the width. This process helps surface exactly why 40% was a good number that was picked (or helps you identify whether or not the 40% is a good number or not, depending on the display sizes).